src/features/components/templates/navigation/template/mobile-menu/mobile-menu.module.scss

```diff
@@ -1,17 +1,15 @@
 @import 'features/styles/theme/variables';
 
-$menu_size: 30.8rem;
-
 .mobile_menu {
     position: fixed;
     inset-block-start: 7rem;
-    inline-size: $menu_size;
+    inline-size: 100%;
     block-size: calc(100vh - 7rem);
     background-color: $color-white;
     opacity: 1;
-    inset-inline-start: -$menu_size;
+    inset-inline-start: -100vh;
     overflow: scroll;
-    transition: inset-inline-start 0.5s ease 0ms;
+    transition:  0.5s ease 0ms;
```

Comment:
Why did you remove `inset-inline-start` ?

Reply:
Previously it was given a width of 30.8rem, but as per new design we are going to have full width. So it's not needed.